pkg/apis/dynatrace/v1alpha1/oneagent_types.go

```diff
@@ -95,11 +95,16 @@ type OneAgentSpec struct {
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Labels"
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:advanced,urn:alm:descriptor:com.tectonic.ui:text"
 	Labels map[string]string `json:"labels,omitempty"`
-	// Optional: set custom proxy settings either directly or from a secret with the field 'proxy'
+	// Optional: Set custom proxy settings either directly or from a secret with the field 'proxy'
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Proxy"
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:advanced,urn:alm:descriptor:com.tectonic.ui:text"
 	Proxy *OneAgentProxy `json:"proxy,omitempty"`
+	// Optional: Adds custom RootCAs from a configmap
```

Comment:
Please explain what happens if not set, i.e., the ones embedded on the image are used, etc.